src/systems/filecoin_markets/deal/_index.md

```diff
@@ -3,6 +3,52 @@ menuTitle: Deals
+    - b. `StorageProvider` calls `HandleStorageDeal` in `StorageMiningSubsystem` which will then add the `StorageDeal` into a `Sector`.
+- 4. Once the miner finishes packing a `Sector`, it generates a Sealed Sector and calls `StorageMinerActor.CommitSector` to verify the seal, store sector expiration, and record the mapping from `SectorNumber` to `SealCommitment`. It will also place this newly added `Sector` in the list of `CommittedSectors` in `StorageMinerActor`. `StorageMiner` does not earn any power for this newly added sector until its first PoSt has been submitted.
+
+# Declare and Recover Faults
+
+- 5. Declared faults are penalized to a smaller degree than spotted faults by `CronActor`. Miners declare faulty sectors by invoking `StorageMinerActor.DeclareFaults` and X of the `StorageDealCollateral` will be slashed and power corresponding to these sectors will be tempororily lost.
+- 6. Miners can then recover faults by invoking `StorageMinerActor.RecoverFaults` and have sufficient `StorageDealCollateral` in their available balances. FaultySectors are recommitted and power is only restored at the next PoSt submission.
+- 7. Sectors that are declared faulty for `storagemining.MaxFaults` consecutive ChainEpochs will result in `StoragePowerActor.SlashPledgeCollateral`.
+  - TODO: set `X` parameter
+
+# Submit PoSt
+
+(TODO: move into Storage Mining)
+
+On every PoSt Submission, the following steps happen.
+
+- 8. `StorageMinerActor` first verifies the PoSt Submission. All Sectors will be considered in `SpottedFaults` if PoSt submission has failed (move to State 14).
+- 9. If `CommittedSectors` are proven in `PoStSubmission.SectorSet`, Storage Miner gains power for these newly committed sectors.
+- 10. If there are `DeclaredFaultySectors` , `Sector` in that set will not be challenged.
+- 11. For all other sectors, payment will be processed by invoking `StorageMarketActor.ProcessStorageDealsPayment` and miner available balances will be updated.
+- 12. Decide which Sectors have expired by looking at the `SectorExpirationQueue`. Sectors expire when all deals in that Sector have expired. `StorageDealCollateral` for both miners and users will only be returned when all deals in the Sector have expired. This is done by calling `StorageMarketActor.SettleExpiredDeals` and the Sector will be deleted from `StorageMinerActor.Sectors`.
+
+# Spot Faults
+
+(TODO: move into Storage Mining)
+
+- 13. `CronActor` calls `StoragePowerActor.EpochTick` at every block. This calls `StorageMinerActor.CheckPoSt` on all the miners whose `ProvingPeriod` is up.
+  - If no PoSt is submitted by the end of the `ProvingPeriod`, `StorageMinerActor` spots the missing PoSt, and sets all sectors to `Failing`.
+    - TODO: reword in terms of a conditional in the mining cycle
+  - When there are sector faults are spotted, both `StorageDealCollateral` and `PledgeCollateral` are slashed, and power is lost.
```

Comment:
As stated, this sounds like a draconian penalty for a single unavoidable failure (e.g. of network or power). I hope there are some missing qualifiers about how much is penalised.